grpc/flatbuffers-java-grpc/src/main/java/com/google/flatbuffers/grpc/FlatbuffersUtils.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
directory-wise, here `main/java` also seems redundant. Since this is a runtime file, it would make sense for me to be stored in the root `java` folder instead, much like grpc.h in C++ is stored with the non-grpc headers.